### PR TITLE
fix: basic id validation

### PR DESCRIPTION
**Pull request by ekremney**
Please ensure your pull request adheres to the following guidelines:
- [ ] make sure to link the related issues in this description
- [ ] when merging / squashing, make sure the fixed issue references are visible in the commits, for easy compilation of release notes

## Related Issues

Thanks for contributing!

### CODE REVIEW

File: src/google-logger.mjs
Comment (ekremney) [inline src/google-logger.mjs:33]: @trieloff I feel like we should use OR instead of AND here

```js
if (!isValidCheckpoint(checkpoint) || !isReasonableWeight(weight) || !isValidId(id)) {
  return;
}
```

Comment (trieloff) [reply src/google-logger.mjs:33]: Yes, I have another PR open for that

**Review by trieloff — APPROVED** [review]

File: src/google-logger.mjs
Comment (trieloff) [reply src/google-logger.mjs:33]: This caused some tests to fail

Comment (ekremney) [reply src/google-logger.mjs:33]: I will wait until that PR is merged

**Pull request merged**
